src/baseTrie.ts

```diff
@@ -62,12 +62,12 @@ export class Trie {
       }
     }
 
-    proofTrie.db.batch(opStack, (e?: Error) => {
-      cb(e, proofTrie)
+    proofTrie.db.batch(opStack, (err) => {
+      cb(err, proofTrie)
```

Comment:
Not really getting this change here, could you elaborate?

Reply:
yes I just renamed `e` to `err` and removed the explicitly defined type (already typed as [`ErrorCallback`](https://github.com/ethereumjs/merkle-patricia-tree/blob/master/src/db.ts#L85))

Reply:
Ah ok, thanks!